bundles/org.openhab.binding.bmwconnecteddrive/README.md

```diff
@@ -0,0 +1,862 @@
+| Hybrid Range Radius   | range            | range-radius-hybrid   | Number:Length        |      |  X   |    X    |     | 
+
+See Description [Range vs Range Radius](#range-vs-range-radius) to get more information
+
+#### Charge Profile
+
+Valid for Electric and Hybrid Vehicles
+These are Read-only values.
+
+| Channel Label                      | Channel Group ID | Channel ID          | Type   | 
+|------------------------------------|------------------|---------------------|--------|
+| Air Conditioning at Departure Time | charge           | profile-climate     | Switch | 
+| Charging Mode for Profile          | charge           | profile-mode        | String | 
+| Charging Window Start Time         | charge           | window-start        | String | 
+| Charging Window End Time           | charge           | window-end          | String | 
+| Timer 1: Departure Time            | charge           | timer1-departure    | String | 
+| Timer 1: Scheduled Days            | charge           | timer1-days         | String | 
+| Timer 1: Enabled                   | charge           | timer1-enabled      | Switch | 
+| Timer 2: Departure Time            | charge           | timer2-departure    | String | 
+| Timer 2: Scheduled Days            | charge           | timer2-days         | String | 
+| Timer 2: Enabled                   | charge           | timer2-enabled      | Switch | 
+| Timer 3: Departure Time            | charge           | timer3-departure    | String | 
+| Timer 3: Scheduled Days            | charge           | timer3-days         | String | 
+| Timer 3: Enabled                   | charge           | timer3-enabled      | Switch | 
+
+
+#### Location
+
+Available for all Vehicles.
+These are Read-only values.
+
+| Channel Label  | Channel Group ID | Channel ID          | Type         | 
+|----------------|------------------|---------------------|--------------|
+| Latitude       | location         | latitude            | Number       | 
+| Longitude      | location         | longitude           | Number       |
+| Heading        | location         | heading             | Number:Angle | 
+
+#### Last Trip
+
+Check [Vehicle Properties](#Properties) if *Statistics* is present in *Services Supported*
+These are Read-only values.
+
+| Channel Label                           | Channel Group ID | Channel ID                   | Type          |
+|-----------------------------------------|------------------|------------------------------|---------------|
+| Last Trip Date                          | last-trip        | date                         | DateTime      |
+| Last Trip Duration                      | last-trip        | duration                     | Number:Time   |
+| Average Power Consumption per 100 km    | last-trip        | average-consumption          | Number:Power  |
+| Average Combined Consumption per 100 km | last-trip        | average-combined-consumption | Number:Volume |
```

Comment:
Just picking a random channel here: Please note that channel labels are expected to be as short as possible. Guideline is 2-3 words with up to 25 chars. Would be great if you could go over the list and check for potential shortening opportunities. :-)

Reply:
Right, some exceeded the limit. Added some abbreviations and checked the channel ids => all below 25 chars now.